content/work/medcurbside.rst

```diff
@@ -0,0 +1,53 @@
+public: yes
+location: case-studies
+pageclasses: work-layout
+grid_template: true
+image:
+  - src: 'projects/medcurbside.jpg'
+client:
+  - name: 'Lab06'
+project:
+  - name: 'MedCurbside'
+    tagline: 'Ask, Answer, Cite. Make Medicine Better.'
+    url: 'https://www.medcurbside.com/'
+    years: '2016–2017'
+    status: 'live'
```

Comment:
This doesn't have `case_study: true`, so we're not linking to this page at all. Should it be `public: no`?

I guess the same question applies to `moztrap` and `quarqrace`.

Reply:
I think this is my only review comment that hasn't been addressed yet. I'm not clear on how we decide when to set `public: yes` vs `public: no`.